competency-explorer/src/helpers/colors.ts

```diff
@@ -0,0 +1,14 @@
+import { Level } from '../models';
+import { getLevelIndex } from './levelDirection';
+
+const MIN_OPACITY = 0.4;
```

Comment:
![Screenshot of light gray headings](https://user-images.githubusercontent.com/3335181/87555329-4d698d80-c683-11ea-94ec-9925990bb2d9.png)

I don't know if this is on your radar / in scope for this app, but these header light grays are below the minimum AA accessibility level for contrast. Maybe having them be italic & a darker gray would look ok?

Similar with the disabled sidebar things; maybe a lighter gray & italic for them would look ok?
![Screenshot of medium gray nav items on a dark gray background](https://user-images.githubusercontent.com/3335181/87555268-3aef5400-c683-11ea-9298-b16aa0398993.png)

Reply:
I think with some of the changes Casey proposed I'll have to rework these headers anyways, so I'll address in the next PR.